checkov/main.py

```diff
@@ -563,11 +565,10 @@ def add_parser_args(parser: ArgumentParser) -> None:
                env_var='CKV_SECRETS_SCAN_FILE_TYPE',
                action='append',
                help='add scan secret for requested files. You can specify this argument multiple times to add '
-                    'multiple file types. To scan all types (".tf", ".hcl", ".yml", ".yaml", ".json", ".template", '
-                    '".bicep", ".py", ".js", ".properties", ".pem", ".php", ".xml", ".ts", ".env", "Dockerfile", '
```

Comment:
@nimrodkor I think it's reverting your earlier fix for hcl and bicep from https://github.com/bridgecrewio/checkov/pull/3472/files

Reply:
But why?

Reply:
fixed